src/sonic-bmpcfgd/setup.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import print_function
+import sys
+from setuptools import setup
+import pkg_resources
+from packaging import version
+
+# sonic_dependencies, version requirement only supports '>='
+sonic_dependencies = ['sonic-py-common', 'sonic-utilities']
+for package in sonic_dependencies:
+    try:
+        package_dist = pkg_resources.get_distribution(package.split(">=")[0])
+    except pkg_resources.DistributionNotFound:
+        print(package + " is not found!", file=sys.stderr)
+        print("Please build and install SONiC python wheels dependencies from sonic-buildimage", file=sys.stderr)
+        exit(1)
+    if ">=" in package:
+        if version.parse(package_dist.version) >= version.parse(package.split(">=")[1]):
+            continue
+        print(package + " version not match!", file=sys.stderr)
+        exit(1)
+
+setup(
+    name = 'sonic-bmpcfgd-services',
+    version = '1.0',
+    description = 'Python services which run in the bmp container',
+    license = 'Apache 2.0',
+    author = 'SONiC Team',
+    author_email = 'linuxnetdev@microsoft.com',
+    url = 'https://github.com/Azure/sonic-buildimage',
+    maintainer = 'Feng Pan',
+    maintainer_email = 'fenpan@microsoft.com',
+    packages = [
+        'host_modules'
+    ],
+    scripts = [
+        'scripts/bmpcfgd'
+    ],
+    install_requires = [
+        'dbus-python',
+        'systemd-python',
+        'Jinja2>=2.10',
+        'PyGObject',
+        'psutil'
```

Comment:
>psutil [](http://example.com/codeflow?start=9&length=6)

Did you use psutil library? Please also check other dependencies are really needed.

Reply:
done